Scheduler/feature_scheduler/maintel/fbs_config_fieldsurvey.py

```diff
@@ -0,0 +1,91 @@
+# This file is part of ts_config_ocs.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.ts.fbs.utils.maintel.basis_functions import get_basis_functions_field_survey
+from lsst.ts.fbs.utils.maintel.detailers import get_detailers_field_survey
+from lsst.ts.fbs.utils.maintel.make_fieldsurvey_scheduler import (
+    MakeFieldSurveyScheduler,
+    get_comcam_sv_targets,
+)
+
+
+def get_scheduler():
+    """Construct feature based scheduler.
+
+    Returns
+    -------
+    nside : int
+        Healpix map resolution.
+    scheduler : Core_scheduler
+        Feature based scheduler.
+    """
+
+    nside = 32
+
+    make_scheduler = MakeFieldSurveyScheduler(nside=nside, ntiers=1)
+
+    nvisits = {"u": 20, "g": 20, "r": 20, "i": 20, "z": 20, "y": 20}
+    sequence = "ugrizy"
```

Comment:
Sequence and all other references to filter names must follow the format f"{band}_{ID}" IE `i_06`, see for example BLOCK-T248. Not doing this will result in script failure on the telescope as it is not valid to set a filter with the band name alone.